controller/telemetry/server.go

```diff
@@ -233,17 +234,17 @@ func (s *server) ListPods(ctx context.Context, req *read.ListPodsRequest) (*publ
 			status = "Terminating"
 		}
 
-		plane, _ := pod.Labels["conduit.io/plane"]
-		controller, _ := pod.Labels["conduit.io/controller"]
+		controllerComponent, _ := pod.Labels[pkgK8s.ControllerComponentLabel]
+		controllerNS, _ := pod.Labels[pkgK8s.ControllerNSLabel]
 
 		item := &public.Pod{
 			Name:                pod.Namespace + "/" + pod.Name,
 			Deployment:          deployment,
 			Status:              status,
 			PodIP:               pod.Status.PodIP,
 			Added:               added,
-			ControllerNamespace: controller,
-			ControlPlane:        plane == "control",
+			ControllerNamespace: controllerNS,
+			ControlPlane:        controllerComponent != "",
```

Comment:
Will this always be empty string if the label is not set?  Could it ever be nil?  Should we defend against that?

Reply:
It can't be nil, per https://golang.org/doc/effective_go.html#maps:
> An attempt to fetch a map value with a key that is not present in the map will return the zero value for the type of the entries in the map.

In this case, zero value of the string will be `""`.

For fun: https://play.golang.org/p/trg0UpaFa_3